### DISCUSSION TITLE
General User Management

### DISCUSSION THREAD

**Discussion by Mekcyed** [TomBursch/kitchenowl#455] [General]
Hey folks,

i have a general question. We have just started using the public Kitchenowl instance via the Android app.
I searched with '@' to add a new household member and found a bunch of users and their email addresses. I think it may not be clear to everyone in the registration process that the usernames are 'public'. Have you thought about doing things differently with household management, such as direct email invitations and not using public usernames?

<img src="https://github.com/TomBursch/kitchenowl/assets/90114325/c00889ba-33c2-4789-b855-091f07fd8272" width="400" height="790">

**Comment by TomBursch** [comment]
Hey, thanks for mentioning this, I will more clearly state that the username and name will be public when registering. Direct email invitations are definitely the better system and something I want to do in the future.
